File: 1603-running-sum-of-1d-array/running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int[] prefix = Arrays.copyOf(nums, nums.length);
        for(int i = 1; i < nums.length; i++){
            prefix[i] += prefix[i - 1];
        }
        
        return prefix;       
    }
}

File: python-track/running-sum-of-1d-array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        ans = [0] * (len(nums))
        ans[0] = nums[0]
        for i in range(1, len(nums)):
            ans[i] = nums[i] + ans[i - 1]
        return ans